lib/GlobalAssemblyInfo.cs

```diff
@@ -32,5 +32,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion( "1.1.0.0" )]
-[assembly: AssemblyFileVersion( "1.1.0.0" )]
+[assembly: AssemblyVersion( "1.1.1.0" )]

```

Comment:
I wasn't sure the process for updating this library, so included it to make sure the version got bumped. Also didn't see a .nuspec file to add my release notes.